email_marketing/media/manage_unsubscriptions

```diff
@@ -0,0 +1,42 @@
+=====================================
+Manage Unsubscriptions and Blacklist
+=====================================
+It is best practice and legally required to allow recipients to manage their state via the
+unsubscription page, as you do not want your audience to think that your company is using any
+tactics that are dishonest or spammy.
+
+Enable the Blacklist feature
+=============================
+Go to :menuselection:`Configuration --> Settings` and enable the option *Blacklist Option when
+unsubscribing*.
+
+
+.. image:: media/manage1.png
+   :align: center
+   :alt: Blacklist in Odoo Email Marketing
+
+
+Now, once the user clicks on the *Unsubscribe* link on your email, he will be redirected to the
+following page:
+
+
+.. image:: media/manage2.png
+   :align: center
+   :alt: Blacklist in Odoo Email Marketing
+
+
+In addition to having the option of unsubscribing from specific mailing lists, the user can also
+blacklist himself, meaning that he will not receive *any* more emails.
+
+
+.. note::
+   The mailing list has to be configured as *Public* in order to be visible for users.
+
+
+Under :menuselection:`Configuration --> Blacklist`, blacklisted email addresses will be shown.
+When opening the record, as a *Log note*, a description-history is kept.
+
+
+.. image:: media/manage3.png
+   :align: center
+   :alt: Blacklist in Odoo Email Marketing
```

Comment:
This shouldn't be in your branch. It's an rst in media, I don't know what happened?